pkg/nuctl/command/import.go

```diff
@@ -0,0 +1,420 @@
+	})
+}
+
+type importFunctionCommandeer struct {
+	*importCommandeer
+	functionConfigs map[string]functionconfig.Config
+}
+
+func newImportFunctionCommandeer(importCommandeer *importCommandeer) *importFunctionCommandeer {
+	commandeer := &importFunctionCommandeer{
+		importCommandeer: importCommandeer,
+	}
+
+	cmd := &cobra.Command{
+		Use:     "function function-config-file",
+		Aliases: []string{"fu"},
+		Short:   "Import function, and by default don't deploy it",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			funcBytes, err := commandeer.readFromStdinOrFile(args)
+			if err != nil {
+				return errors.Wrap(err, "Failed to read function data")
+			}
+
+			// initialize root
+			if err := importCommandeer.rootCommandeer.initialize(); err != nil {
+				return errors.Wrap(err, "Failed to initialize root")
+			}
+
+			unmarshalFunc := commandeer.getUnmarshalFunc(commandeer.format)
+
+			if commandeer.multiple {
+				err = commandeer.parseMultipleFunctionsImport(funcBytes, &commandeer.functionConfigs, unmarshalFunc)
+			} else {
+				commandeer.functionConfigs = map[string]functionconfig.Config{}
+				err = commandeer.parseFunctionImport(funcBytes, &commandeer.functionConfigs, unmarshalFunc)
+			}
+			if err != nil {
+				return errors.Wrap(err, "Failed to parse function data")
+			}
+
+			return commandeer.importFunctions(commandeer.functionConfigs, commandeer.deploy, "")
+		},
+	}
+
+	commandeer.addImportCommandFlags(cmd)
+
+	commandeer.cmd = cmd
+
+	return commandeer
+}
+
+func (i *importFunctionCommandeer) parseFunctionImport(funcBytes []byte,
+	functionConfigs *map[string]functionconfig.Config,
+	unmarshalFunc func(data []byte, v interface{}) error) error {
+
+	functionConfig := &functionconfig.Config{}
+	if err := unmarshalFunc(funcBytes, functionConfig); err != nil {
+		return errors.Wrap(err, "Failed encoding function import config")
+	}
+
+	(*functionConfigs)[functionConfig.Meta.Name] = *functionConfig
+
+	return nil
+}
+
+func (i *importFunctionCommandeer) parseMultipleFunctionsImport(funcBytes []byte,
+	functionConfigs *map[string]functionconfig.Config,
+	unmarshalFunc func(data []byte, v interface{}) error) error {
+	if err := unmarshalFunc(funcBytes, functionConfigs); err != nil {
+		return errors.Wrap(err, "Failed encoding function import config")
+	}
+
+	return nil
+}
+
+type projectImportConfig struct {
+	Project        platform.ProjectConfig
+	Functions      map[string]functionconfig.Config
+	FunctionEvents map[string]platform.FunctionEventConfig
+}
+
+type importProjectCommandeer struct {
+	*importCommandeer
+	projectImportConfigs map[string]projectImportConfig
+}
+
+func newImportProjectCommandeer(importCommandeer *importCommandeer) *importProjectCommandeer {
+	commandeer := &importProjectCommandeer{
+		importCommandeer: importCommandeer,
+	}
+
+	cmd := &cobra.Command{
+		Use:     "project project-config-file",
+		Aliases: []string{"proj"},
+		Short:   "Import project, all it's functions and functionEvents",
```

Comment:
```suggestion
		Short:   "Import project and all its functions and function events",
```

Reply:
You are correct about the apostrophe, but changing the comma to 'and' is grammatically incorrect.